server/redis.ts

```diff
@@ -4,6 +4,7 @@ import redis from "redis";
 import env from "./env";
 
 const client = redis.createClient({
+  url: env.REDIS_URL,
```

Comment:
The short-circuit syntax `REDIS_URL || {}` is not allowed with the `@types/redis` type definition.

```
       server/redis.ts(6,35): error TS2769: No overload matches this call.
         The last overload gave the following error.
           Argument of type 'string | { password: string; host: string; port: number; }' is not assignable to parameter of type 'ClientOpts'.
             Type 'string' has no properties in common with type 'ClientOpts'.
```